todo_app/trello_items.py

```diff
@@ -2,6 +2,8 @@
 import requests
```

Comment:
The name of this file is now confusing, as it has nothing to do with Trello any more! It would be good to update that